components/form/Input.js

```diff
@@ -13,17 +24,17 @@ const Input = ({ className, Icon, label, name, placeholder, type, register, erro
         type={type}
         id={name}
         {...register}
-        className={`${errorClass} block py-2.5 px-0 w-full text-sm text-amcovad-secondary-700 bg-transparent placeholder-transparent border-2 rounded-md border-amcovad-secondary-300 appearance-none focus:outline-none focus:ring-0 focus:border-amcovad-primary-500 peer ${className}`}
+        className={`${errorClass} block py-2.5 px-0 w-full text-sm  text-amcovad-secondary-700 bg-transparent placeholder-transparent border-2 rounded-md border-amcovad-secondary-300 appearance-none focus:outline-none focus:ring-0  peer ${className}`}
       />
       <label
         htmlFor={name}
-        className={`absolute text-sm text-amcovad-secondary-700 duration-300 transform -translate-y-7 scale-75 top-3 left-1 z-10 origin-[0] peer-focus:left-1 peer-focus:text-amcovad-primary-500 peer-placeholder-shown:scale-100 peer-placeholder-shown:translate-y-0 peer-focus:scale-75 peer-focus:-translate-y-7`}
+        className={`${errorLabel} absolute text-sm text-amcovad-secondary-700 duration-300 transform -translate-y-7 scale-75 top-3 left-1 z-10 origin-[0] peer-focus:left-1  peer-placeholder-shown:scale-100 peer-placeholder-shown:translate-y-0 peer-focus:scale-75 peer-focus:-translate-y-7`}
       >
         {label}
       </label>
       {Icon && (
         <span className="absolute top-4 right-2 cursor-pointer" data-testid="icon">
-          {Icon}
+          {Icon ? HasIcon : ''}
```

Comment:
kindly help review this getting [object object] error

Reply:
Your implementation here is highly confusing, let's discuss this in today's sync.